src/base.py

```diff
@@ -2,122 +2,80 @@
 from fastai.vision import *
 from fastai.callbacks import *
 from fastai.vision.gan import *
-from PIL import Image, ImageDraw, ImageFont
-import matplotlib.pyplot as plt
 import os
-import random as rand
 from pathlib import Path
-from wmtransformer import oldMold
-
-def checkpt(ep, lr, model, interval, mod_name='Model_'):
-    cpi = round(ep / interval)
-    for z in range(cpi):
-        model.fit(interval, lr)
-        model.save(mod_name + str(z+1))
-
-# The Following functions (and model is inspired) from: https://tinyurl.com/yybfbbg4
-def get_data(bs, size):
-    data = (src.label_from_func(lambda x: path_hr / x.name)
-            .transform(get_transforms(max_zoom=2.), size=size, tfm_y=True)
-            .databunch(bs=bs).normalize(imagenet_stats, do_y=True))
-    data.c = 3
-    return data
-
-def get_crit_data(classes, bs, size):
-    src = ImageList.from_folder(path, include=classes).split_by_rand_pct(0.1, seed=42)
-    ll = src.label_from_folder(classes=classes)
-    data = (ll.transform(get_transforms(max_zoom=2.), size=size)
-            .databunch(bs=bs).normalize(imagenet_stats))
-    data.c = 3
-    return data
-
-def save_preds(dl):
-    i = 0
-    names = dl.dataset.items
-    for b in dl:
-        preds = model_gen.pred_batch(batch=b, reconstruct=True)
-        for o in preds:
-            o.save(path_gen / names[i].name)
-            i += 1
-
-def create_gen_learner(pretrain=True):
-    # Loading resnet34 into unet to pretrain
-    if pretrain is True:
-        arch = models.resnet34
-        #arch()
-    return unet_learner(data_gen, arch, wd=1e-3, blur=True, norm_type=NormType.Weight,
-                        self_attention=True, y_range=(-3., 3.), loss_func=MSELossFlat())
-
-def create_critic_learner(data, metrics):
-    # Assigning loss function for Critic and creating the critic Model
-    loss_critic = AdaptiveLoss(nn.BCEWithLogitsLoss())
-    return Learner(data, gan_critic(), metrics=metrics, loss_func=loss_critic, wd=1e-3)
-
-
-# ASSIGNING PATHS - CREATING DIRs
-path = Path('/data/')
-path_hr = path / 'raw'
+from wmtransformer import *
+from model_utils import *
+
+# Assigning default paths
+path = Path('AI-for-Blemish-Detection-and-Repair/data/')
```

Comment:
Relative path here.

Reply:
I've done this already a couple days after this pull request, this pull request is behind on the branch.